submissions/VBystrov/dom/js/app.js

```diff
@@ -0,0 +1,50 @@
+import creatures from './tower.js';
+
+document.addEventListener('DOMContentLoaded', () => {
+  const navigation = document.getElementById('navigation');
+  const main = document.getElementById('main');
+  const buttonText = document.getElementById('button-text');
+
+  navigation.addEventListener('click', (e) => {
+    if (e.target.classList.contains('nav-item')) {
+      buttonText.textContent = e.target.textContent;
+
+      // main.innerHTML = '';
+      let desc = main.getElementsByClassName('description');
```

Comment:
Use `const` here instead of `let`, please. You didn't reassign this variable in your further actions in code.

Reply:
Done.